src/app/store.js

```diff
@@ -35,9 +35,12 @@ if (isDemo) {
         user: demoUserObject,
         channels: new Array(randomNumber(200, 600)).fill({
             isDM: true,
+            guild: {
+                name: 'Awesome Discord Server'
+            },
             messages: [demoMessageObject]
         }),
-        guilds: new Array(randomNumber(10, 100)).fill({ id: 123456789, name: "Demo" }),
+        guilds: new Array(randomNumber(10, 100)).length,
```

Comment:
guilds is expected to be an array. the length property is used in the Home.svelte file